src/app.py

```diff
@@ -63,6 +65,13 @@ def uploadpage():
         print("api offline")
     return flask.render_template('upload.html', title='Devcade - Upload', gamelist=usergames)
 
+@app.route('/download/<id>')
+def download(id):
+    r = requests.get(app.config["DEVCADE_API_URI"] + "games/download/" + id, stream=True)
+    b = BytesIO(r.content)
```

Comment:
Am not a fan of the single character variable names. Especially as we focus on refactoring and documentation for a bit after makerfair

Reply:
We can easily document the code without changing var names.  I don't see what's wrong with keeping them short when it's standard to keep them as simple as possible, especially when they are intuitive as is.  May I suggest a documentation pr?